### PR TITLE
fix: remove extra 'backtrace:' when backtrace support isn't compiled in

### PR DESCRIPTION
**Pull request by iacore**

### CODE REVIEW

File: src/runtime/object.cpp [hunk 1]
```diff
@@ -100,13 +101,10 @@ extern "C" LEAN_EXPORT object * lean_panic_fn(object * default_val, object * msg
     // TODO(Leo, Kha): add thread local buffer for interpreter.
     if (g_panic_messages) {
         std::cerr << lean_string_cstr(msg) << "\n";
-#ifdef __GLIBC__
         char * bt_env = getenv("LEAN_BACKTRACE");
         if (!bt_env || strcmp(bt_env, "0") != 0) {
-            std::cerr << "backtrace:\n";
```
Comment (Kha) [inline src/runtime/object.cpp:106]: This line clearly is not executed for `LEAN_BACKTRACE=0`...?

**Comment by Kha** [comment]
How is this supposed to change anything,

File: src/runtime/object.cpp (hunk 1 above)
Comment (locriacyber) [reply src/runtime/object.cpp:106]: sorry, wrong description

Comment (Kha) [reply src/runtime/object.cpp:106]: This line also clearly is not executed when `__GLIBC__` is not defined...

**Pull request closed without merging**